mixins/visible-on-ancestor/visible-on-ancestor-mixin.js

```diff
@@ -15,6 +15,17 @@ export const visibleOnAncestorStyles = css`
 		transition: transform 200ms ease-out, opacity 200ms ease-out !important;
 	}
 
+	:host([__voa-state="hidden"][animation-type="opacity"]),
+	:host([__voa-state="hiding"][animation-type="opacity"]) {
+		transform: none !important;
```

Comment:
It's too bad we need all these `!important` declarations. I know that the original code is basically forcing you to follow suit with this. I have a sneaking suspicion that the original reason for them was to trump the mixin consumer if they tried to set these properties in their CSS. 🤷‍♂️ Do you see another reason why we would have used them?

Reply:
That was my guess as well. I don't know of/didn't encounter another reason.